### PR TITLE
[wmco] Remove stuttering from instances pkg

### PR DESCRIPTION
**Pull request by aravindhp**
- Rename instances pkg to instance
- Rename InstanceInfo to Info
- Rename NewInstanceInfo to NewInfo

This results in more natural names for the types and functions in their call sites.

### CODE REVIEW

**Review by jrvaldes — APPROVED** [review]
/lgtm

File: pkg/instance/instance.go [hunk 1]
```diff
@@ -19,14 +19,14 @@ type InstanceInfo struct {
 	Node *core.Node
 }
 
-// NewInstanceInfo returns a new instanceInfo. newHostname being set means that the instance's hostname should be
+// NewInfo returns a new instanceInfo. newHostname being set means that the instance's hostname should be
```
Comment (sebsoto) [inline pkg/instance/instance.go:22]: `new instanceInfo` -> `new Info`

**Review by sebsoto — CHANGES_REQUESTED** [review]

**Comment by aravindhp** [comment]
/hold

until https://github.com/openshift/release/pull/21004 merges

**Comment by aravindhp** [comment]
/hold cancel

https://github.com/openshift/release/pull/21004 has merged

/retest

**Comment by aravindhp** [comment]
/hold
until we bump the kebelet sub-module to 1.22-rc.0

**Comment by aravindhp** [comment]
Seeing a [flake](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/572/pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator/1426165308594327552) with HNS deletions.

**Pull request merged**